### PR TITLE
Install and configure VMware vCenter

### PR DESCRIPTION
**Pull request by MasterWayZ**
This is a tutorial that I wrote on how to deploy the vCenter Server Appliance, configuring it, creating your first Datacenter and adding your first host.

It has a link to a tutorial on how to install ESXi, but it's a placeholder link. There is a pull request open for it (made by someone else).

I have read and understood the Contributor's Certificate of Origin available at the end of 
https://raw.githubusercontent.com/hetzneronline/community-content/master/tutorial-template.md
and I hereby certify that I meet the contribution criteria described in it.
Signed-off-by: Michael Goossens <michaelgoossens@live.be>

PS: the last commit was made by "Michelle Goossens" which is me (Michael / MasterWayZ), I made an error and had to git clone the repo again and forgot to edit the user.name git config variable.

### CODE REVIEW

File: tutorials/install-and-configure-vmware-vcenter/01.en.md [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+(c) The contribution was provided directly to me by some other person
+    who certified (a), (b) or (c) and I have not modified it.
+
+(d) I understand and agree that this project and the contribution are
+    public and that a record of the contribution (including all personal
+    information I submit with it, including my sign-off) is maintained
+    indefinitely and may be redistributed consistent with this project
+    or the license(s) involved.
+
+Signed-off-by: [ Michael Goossens ( michaelgoossens@live.be )]
```
Comment (n-se) [inline tutorials/install-and-configure-vmware-vcenter/01.en.md]:
```suggestion
Signed-off-by: Michael Goossens <michaelgoossens@live.be>
```

**Comment by MasterWayZ** [comment]
I have added the commits and added a commit from myself, which fixed the spelling errors that I was able to find.

**Comment by leojakeson** [comment]
This looks good. I've made some small typo and formatting updates.

Before I merge it, I think it would be great if you could add some screenshots. Right now the tutorial is a wall of text, with nothing to break that up. IMHO a few screenshots would greatly help both to show what is happening, and to improve the readability.

**Comment by MasterWayZ** [comment]
Are the screenshots allowed to be from a setup following the tutorial but not on a Hetzner server? (There isn't any difference, so it shouldn't be a problem)

**Comment by leojakeson** [comment]
That's not a problem... as long as there aren't any logos/references to any of our (major) competitors :)

**Comment by MasterWayZ** [comment]
Okay, thank you! That won't be a problem, don't worry!

I'll prepare a lab for it at home so I can make some screenshots of the installation, once I've done that I'll add them.

**Comment by MasterWayZ** [comment]
My apologies for the delay. I will get to this in a few days, I have been busy.

**Comment by MasterWayZ** [comment]
I have added some screenshots for the first step. For which other steps would you like to have screenshots added for?

**Comment by n-se** [comment]
Step number 3 is also a quite big wall of text and it would be much easier to follow if you add some screenshots there two. Would that be possible? :)

**Comment by MasterWayZ** [comment]
This is possible, I will try to get screenshots as soon as I can. I'll also note for further tutorials that I will try to include screenshots when I can 😃

**Comment by MasterWayZ** [comment]
Once again my apologies for the long wait. My lab server crashed so I couldn't make the screenshots. It's now working again and tomorrow I'm getting the drive that I can install vCenter on so I can take more screenshots.

**Comment by n-se** [comment]
No worries, take your time :)

**Comment by MasterWayZ** [comment]
I accidentally made a push that merged the upstream version into this and it made a commit. I tried reverting it but I think that I only made it worse, sorry. I hope that you can fix it.

I have just pushed out a new version with screenshots on most steps. Please let me know if there is anything that you would like me to add.

**Comment by MasterWayZ** [comment]
I also have a question:
There's currently an existing PR for installing vSphere on Hetzner servers, however it has not been updated since March. What is going to happen with it?

Also once again, I've updated the PR with screenshots. Does anything else need to be added?

**Pull request merged**